rebar.lock

```diff
@@ -1 +1,4 @@
-[{<<"lfe">>,{pkg,<<"lfe">>,<<"1.0.2">>},0}].
+[{<<"lfe">>,
```

Comment:
We can't do this. Hex packages must have all Hex packages as dependencies. There's also no reason not to us the LFE hex package, as it has no dependencies and involves no additional effort by the user beyond installing `rebar3`.